Aero_Sub_DAQ/Air_pressure_sensor_code/Air_pressure_sensor_code.ino

```diff
@@ -0,0 +1,129 @@
+#include <Wire.h>
+#define P1 0x29
```

Comment:
What is P1?

- [ ] More explicit names

Reply:
p1 is the address when we call Wire.beginTransmission(P1)